src/app/shared/models/icon.enum.ts

```diff
@@ -4,9 +4,11 @@
 export enum Icon {
     Quiddity1,
     Quiddity2,
-    Quiddity3,
     Draw1,
     Draw2,
     ReRoll,
-    Ready
+    Ready,
+    Quiddity3OrReRoll,
```

Comment:
@phyxl would you rather these three compound icons be treated as their own value or be added to the dieside as an array?

Reply:
It's best as an array I think, to avoid repeating ourselves

Reply:
we'll need a bool to decide weather the array is or/and